packages/jellyfish-api-core/src/category/misc.ts

```diff
@@ -1,4 +1,5 @@
 import { ApiClient } from '..'
+import { wallet } from '@defichain/jellyfish-api-core'
```

Comment:
```suggestion
import { wallet } from '../???'
```

Best use relative path for internal linking.